modules/claims_api/lib/bgs_service/corporate_update_web_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class CorporateUpdateWebService < ClaimsApi::LocalBGS
+    def bean_name
+      'CorporateUpdateServiceBean'
```

Comment:
Sorry for the piecemeal review, but should `bean_name` be:

```suggestion
      'CorporateUpdateServiceBean/CorporateUpdateWebService'
```

? Iʼm having trouble calling the service from the console.

Reply:
Realizing now that `def bean_name` in all our services is a bit of misnomer. Iʼm thinking something like `def endpoint` would be more appropriate 😅

Reply:
> Sorry for the piecemeal review
No worries, I appreciate you.

Updated for consistency with other services. Tested calls to the service via PoaVBMSUpdater, and oddly it works either way, even though `bean_name` is used in LocalBGS.

Agree bean_name is a misnomer; guessing that's a vestige of the bgs-ext [implementation](https://github.com/department-of-veterans-affairs/bgs-ext/blob/350e45ae699407948d254d6fadd4f1b249e996f5/lib/bgs/services/corporate_update.rb#L15). Could clean that up in [API-42308](https://jira.devops.va.gov/browse/API-42308) as well.

Reply:
Oh strange, it is working either way – I mustʼve been hitting the breaker yesterday. I agree, we could do a rename for consistency in the cleanup task. The wonders of BGS!